specification/dimensions/billingperiodend.md

```diff
@@ -0,0 +1,32 @@
+# Billing Period End
+
+Billing period represents the time window for which you have or will receive an invoice for. The time window is inclusive of the start date and exclusive of the end date.
+
+Billing Period End represents the end of a billing period.
```

Comment:
We say it's the "end" here, but it's actually not. It's actually the beginning of the next billing period since the value is exclusive. Should we be explicit about that or give an example?

Reply:
but semantically, its the end time because it is not inclusive of it. So if it was showing YYYY-mm-ddT02:00:00.000Z or whatever, the semantics for that column says it exclusive of that time which really means 01:59:59.999Z right?